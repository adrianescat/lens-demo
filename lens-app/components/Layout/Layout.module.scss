.layout {
  display: flex;
  text-rendering: optimizeLegibility;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-smoothing: antialiased;
  flex-direction: column;
  font-family: 'MSQT';
  height: 100%;
}