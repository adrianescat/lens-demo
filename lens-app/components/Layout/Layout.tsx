import React, { ReactElement } from 'react';
import styles from './Layout.module.scss';

export default function Layout({ children }): ReactElement {
  return <div className={styles.layout}>{children}</div>
}