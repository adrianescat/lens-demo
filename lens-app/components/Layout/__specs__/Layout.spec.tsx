import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../Layout';

let wrapper: any;
let props: any;

beforeEach(() => {
  props = { children: <div>Test</div> };
  wrapper = shallow(<Layout {...props} />);
});

test('should match snapshots', () => {
  expect(wrapper).toMatchSnapshot();
});