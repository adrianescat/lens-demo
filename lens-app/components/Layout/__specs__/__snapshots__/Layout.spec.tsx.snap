// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshots 1`] = `
<div
  className="layout"
>
  <div>
    Test
  </div>
</div>
`;
